test/user.test.ts

```diff
@@ -8,22 +8,16 @@ const request = require('supertest')(url);
 
 describe('User endpoint', async function () {
   let connection: Connection;
+  let userRepository: Repository<UserEntity>;
 
   before(async () => {
     connection = getConnection();
+    userRepository = connection.getRepository(UserEntity);
     await connection.synchronize();
```

Comment:
O synchronize é uma operação que teoricamente só precisa ser feita quando existe alguma troca estrutural no banco, né? Adição/alteração de coluna ou tabela, por exemplo. Mas por simplicidade aqui nesse projetinho, talvez vc possa deixar ele logo após a connection lá no `startDatabaseConnection`. A solução que adotar verificar nos projetos você pode dar uma estudada mais pra frente, após o final do onboard.

Deixar no before de cada teste vai fazer com que ele rode desnecessariamente no início de cada fluxo de testes, certo?